helpdesk_mgmt_timesheet_report_order/models/account_analytic_line.py

```diff
@@ -0,0 +1,17 @@
+# © 2021 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See https://www.gnu.org/licenses/lgpl-3.0.html
+from odoo import fields, models, api
+
+
+class AccountAnalyticLine(models.Model):
+    _inherit = "account.analytic.line"
+
+    @api.multi
+    def report_sorted(self):
```

Comment:
More specific naming:
```suggestion
    def report_sorted_by_ticket_priority(self):
```